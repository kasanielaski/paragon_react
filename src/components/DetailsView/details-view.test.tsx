import { describe, it, expect } from 'vitest';
import { render } from '@testing-library/react';

import { DetailsView } from '../index';

const props = {
  formattedPrice: '',
  handleGoBack: () => {}
};

describe('DetailsView component', () => {
  it('should match snapshot', () => {
    const { asFragment } = render(<DetailsView {...props} />);
    expect(asFragment()).toMatchSnapshot();
  });
});
