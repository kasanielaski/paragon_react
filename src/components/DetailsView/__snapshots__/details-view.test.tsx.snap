// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`DetailsView component > should match snapshot 1`] = `
<DocumentFragment>
  <div
    class="sc-fLDLck kxCmhJ"
  >
    <picture
      class="sc-dycHyt xCvTA"
    >
      <img />
    </picture>
    <span />
    <span
      style="text-align: center;"
    />
    <span>
      Price: 
    </span>
    <div
      class="sc-jbAkgO gpPZIG"
    >
      <button>
        Go back
      </button>
    </div>
  </div>
</DocumentFragment>
`;
